src/citrine/resources/predictor.py

```diff
@@ -372,19 +398,11 @@ def create_default(self,
         data = self.session.post_resource(path, json=body, version=self._api_version)
         return self.build(Predictor.wrap_instance(data["instance"]))
 
-    @deprecated(deprecated_in="1.47.0", removed_in="2.0.0",
```

Comment:
Given this is version 1.50, are we killing this too early (based on the removed_in)?

Reply:
I just moved it further down in the file, to group all the deprecated methods together.

Reply:
Gotcha